src/hooks/usePreloadImages.ts

```diff
@@ -0,0 +1,31 @@
+import { useLayoutEffect, useState } from 'react';
+interface ImagePreloadOptions {
+  baseURL: string;
+  imageUrls: string[];
+}
+const usePreloadImages = ({
+  baseURL,
+  imageUrls,
+}: ImagePreloadOptions): boolean => {
```

Comment:
baseURL이 변경 될 상황이 있나요? `import.meta.env.VITE_IMG_BASE_URL`에서 변경 될 상황이 없으면 baseURL Props는 굳이 필요 없이 내부적으로 처리 해도될 것 같습니다.
만약, 변경할 상황이 있다면 props는 유지하되, 기본 값을 설정하면 좋을 것 같습니다.

```suggestion
const IMG_DEFAULT_BASE_URL = import.meta.env.VITE_IMG_BASE_URL;

const usePreloadImages = ({
  baseURL = IMG_DEFAULT_BASE_URL,
  imageUrls,
}: ImagePreloadOptions): boolean => {}
```

Reply:
다른 프로젝트면 모르겠지만 현재 프로젝트에서는 baseURL이 변경 될 상황이 거의 없다보니 @ssi02014 님 의견대로
기본 값을 설정해주면 사용할때 더 좋을 것 같네요!